packages/office-ui-fabric-react/src/components/Layer/Layer.notification.ts

```diff
@@ -7,8 +7,8 @@ let _defaultHostSelector: string | undefined;
 
 /**
  * Register a layer for a given host id
- * @param hostId Id of the layer host
- * @param layer Layer instance
+ * @param hostId - Id of the layer host
+ * @param layer - Layer instance
```

Comment:
Looks like this doc comment was wrong before but the second param of this function is actually `callback`.